Source/Scene/Model.js

```diff
@@ -556,8 +556,9 @@ define([
         this._loadError = undefined;
         this._loadResources = undefined;
 
-        this._perNodeShowDirty = false;             // true when the Cesium API was used to change a node's show property
+        this._perNodeShowDirty = false;            // true when the Cesium API was used to change a node's show property
         this._cesiumAnimationsDirty = false;       // true when the Cesium API, not a glTF animation, changed a node transform
+        this._dirty = false;                       // true when the model was transformed this frame
```

Comment:
Add a `@private` property getter for this.  Otherwise, it looks like this is not used.